app/src/main/java/de/westnordost/streetcomplete/quests/max_weight/Measure.kt

```diff
@@ -0,0 +1,32 @@
+package de.westnordost.streetcomplete.quests.max_weight
+
+//if the minimum required API would be 24, we could maybe use https://developer.android.com/reference/android/icu/util/Measure
+abstract class Measure {
+    abstract fun toTons(): Double
+    abstract override fun toString(): String
+}
```

Comment:
Maybe rename to `WeightMeasure`, as there is already another Measure-class?

Also, this is a good example of a candidate for a `sealed` class.

Reply:
I changed also Measure in max height quest.